.github/workflows/java_ci.yml

```diff
@@ -56,4 +56,12 @@ jobs:
         uses: gradle/actions/setup-gradle@v4
 
       - name: Execute Gradle build
-        run: ./gradlew build
+        run: |
+          ./gradlew build
+          ./gradlew test
+
+      - name: Upload test report
+      - uses: qltysh/qlty-action/coverage@v2
+        with:
+          token: ${{ secrets.QLTY_COVERAGE_TOKEN }}
```

Comment:
Is this token already set in our env?

Reply:
I created the secret variable but with a placeholder since I have no permission on the project in qlty. I will check as this will work.